src/test/java/christmas/ApplicationTest.java

```diff
@@ -41,10 +68,29 @@ class ApplicationTest extends NsTest {
         });
     }
 
-    @Test
-    void 주문_예외_테스트() {
+    @ValueSource(strings = {"0" , "32" , "341"})
+    @ParameterizedTest
+    void 날짜_예외_테스트_범위를_벗어나는_값_입력(String date) {
+        assertSimpleTest(() -> {
+            runException(date);
+            assertThat(output()).contains("[ERROR] 유효하지 않은 날짜입니다. 다시 입력해 주세요.");
```

Comment:
```suggestion
assertThat(output()).contains(INVALID_DATE.getMessage());
```

만약, 에러메세지를 수정한다면, 관련 시스템 테스트 메세지도 전부 수정해야하는 문제가 발생할 것 같아, 위와 같은 의견을 드립니다.